schema-guide.md

```diff
@@ -313,87 +314,93 @@ Array of [`definitions.reference`](#definitionsreference) objects.
 
 ### `repository`
 
-- type: ...
+- type: [`definitions.url`](#definitionsurl)
 - required: false
 
-...
+The URL of the software or dataset in a repository (when the repository is neither a source code repository nor a build artifact repository).
 
 Usage example:
 
 ```yaml
+repository: "https://ascl.net/2105.013"
 ```
 
 ### `repository-artifact`
 
-- type: ...
+- type: [`definitions.url`](#definitionsurl)
 - required: false
 
-...
+The URL of the software in a build artifact/binary repository.
 
 Usage example:
 
 ```yaml
+repository-artifact: "https://search.maven.org/artifact/org.corpus-tools/cff-maven-plugin/0.4.0/maven-plugin"
 ```
 
 ### `repository-code`
 
-- type: ...
+- type: [`definitions.url`](#definitionsurl)
 - required: false
 
-...
+The URL of the software in a source code repository.
 
 Usage example:
 
 ```yaml
+repository-code: "https://github.com/citation-file-format/cff-converter-python"
 ```
 
 ### `title`
 
-- type: ...
+- type: string
 - required: true
 
-...
+The name of the software or dataset.
 
 Usage example:
 
 ```yaml
+title: "cffconvert"
 ```
 
 ### `type`
 
-- type: ...
+- type: enum (`"software"` or `"dataset"`)
+- default: `"software"`
```

Comment:
```suggestion
- type: enum (`software` or `dataset`)
- default: `software`
```

Can we do without the quotes? (Also in the example)

Reply:
Sure can do, although this obfuscates the string-ity a tad, but that's fine with me (and any confusion is entirely YAML's fault).